VideoLocker/res/layout/discussion_responses_thread_row.xml

```diff
@@ -37,6 +37,15 @@
             android:textSize="@dimen/edx_small"
             tools:text="Thread body" />
 
+        <TextView
+            android:id="@+id/discussion_responses_thread_row_visibility_text_view"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/discussion_responses_standard_margin"
```

Comment:
yeah, looking at the mockup and the style guide again, there doesn't seem to no margin or so. But iOS has it. So what do we do ?